examples/mensuration_and_polling/README.md

```diff
@@ -0,0 +1,71 @@
+# Mensuration with Label Studio
+
+![screenshot](data/Screenshot%20from%202024-06-14%2009-18-21.png)
+
+Label georeferenced images in Label Studio, and use the SDK to perform [mensuration](https://pro.arcgis.com/en/pro-app/latest/arcpy/image-analyst/mensuration-class.htm) (measure real-world distances within the image) using the labels produced.
+
+In this example, the [Label Studio SDK](https://labelstud.io/sdk/index.html) is run in a separately running background task that watches for new annotations, calculates the desired distances for each one, and adds the distance to the annotation as a new field.
+
+You will learn:
+
+- How to download and work with satellite imagery
+- How to label polygons in Label Studio
+- How to compute distances in georeferenced images
+- How to use the Label Studio SDK to poll for new annotations and modify task data (this is a toy example - for production use cases, we recommend using Custom Scripts in Label Studio Enterprise for real-time updates to the labeling interface on [annotation events](https://labelstud.io/guide/frontend_reference#Annotation-events).)
+
+
+## Setup data
+
+Any georeferenced raster can be used. For this demo, a georeferenced TIFF image of the Grand Canyon and a corresponding PNG image that can be loaded into Label Studio (TIFF is not supported in Label Studio) was generated using the following steps:
+
+1. Sign up at sentinel-hub.com for a trial account for access to [Sentinel-2](https://en.wikipedia.org/wiki/Sentinel-2) satellite images on demand. These have a resolution of 10 GSD (10 meters per pixel).
+2. Use the included `grab_georeferenced_image.py` with your credentials to download the image in a [UTM coordinate system](https://en.wikipedia.org/wiki/Universal_Transverse_Mercator_coordinate_system), where each pixel is a fixed area instead of a fixed fraction of longitude and latitude. This makes the image "square" with respect to the ground.
+3. Export to PNG:
+```bash
+convert data/e9b9661bcbd97b67f45364aafd82f9d6/response.tiff data/e9b9661bcbd97b67f45364aafd82f9d6/response.png
+```
+
+## Load data into Label Studio
+
+Create a new project and upload `data/e9b9661bcbd97b67f45364aafd82f9d6/response.png` to the project.
```

Comment:
can it be more readable path?

Reply:
this is how the data is returned from sentinelhub along with the response.json, so it's also kind of a tutorial on how to do that. Can flatten the path if we don't care about how getting the imagery works.

Reply:
done